extlinks/links/admin.py

```diff
@@ -19,6 +27,12 @@ class LinkSearchTotalAdmin(admin.ModelAdmin):
 
 class LinkEventAdmin(admin.ModelAdmin):
     list_display = ("link", "timestamp", "domain", "username", "change")
+    readonly_fields = ["url_pattern_display"]
+    exclude = ["content_type", "object_id"]
```

Comment:
We might want to add `raw_id_fields` for the `url` field like explained here: https://docs.djangoproject.com/en/4.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.raw_id_fields. Maybe this will also improve rendering

Reply:
I just tested adding `raw_id_fields = ["url"]` and it's blank. I suggest we add `url` to the `exclude` array